docs/manual/common/guide/devmode/CassandraServer.md

```diff
@@ -63,11 +63,11 @@ In Maven, you can do this by modifying the service implementations pom configura
 
 In sbt, let's assume you have the following Lagom project in your build:
 
-@[cassandra-users-project](code/build-cassandra-opts.sbt)
+@[cassandra-users-project](code/build-cassandra-opts-lang-dependant.sbt)
```

Comment:
This is a clever trick! I didn't realize it would work. I think we have other pages where the only difference between Java and Scala is in the snippet. We could use this trick in those pages, too. I don't think we should go through and change them now, but it's something to keep in mind as we work on the docs.

My only comment is that the filename should be "dependent" rather than "dependant". I'll see if I can push a fix to your branch, or else fix it up after the fact.